packages/teuchos/CMakeLists.txt

```diff
@@ -148,13 +148,11 @@ TRIBITS_ADD_OPTION_AND_DEFINE(
 # ${PACKAGE_NAME}_ENABLE_COMPLEX_FLOAT and
 # ${PACKAGE_NAME}_ENABLE_COMPLEX_DOUBLE.  See Trilinos GitHub #362.
 
-ASSERT_DEFINED(${PROJECT_NAME}_ENABLE_CXX11)
-SET(${PACKAGE_NAME}_ENABLE_LONG_LONG_INT_DEFAULT ${${PROJECT_NAME}_ENABLE_CXX11})
 TRIBITS_ADD_OPTION_AND_DEFINE(
   ${PACKAGE_NAME}_ENABLE_LONG_LONG_INT
   HAVE_TEUCHOS_LONG_LONG_INT
-  "Enable Teuchos features for the 'long long' data type.  This is ON by default if and only if C++11 is enabled.  Please remember that 'long long' only entered the C++ language standard with C++11, even though many compilers supported it before."
-  ${${PACKAGE_NAME}_ENABLE_LONG_LONG_INT_DEFAULT}
+  "Enable Teuchos features for the 'long long' data type."
+  ON
   )
```

Comment:
Two things:
  1. Could the documentation string also say: "This option is DEPRECATED.  Teuchos will ignore this option.  Teuchos requires C++11, and long long comes with C++11, so you will always get long long.  If you want to change the set of global ordinal types that Tpetra enables, set a Tpetra option, not a Teuchos option."
  2. Could we add a warning after this if users set it to `OFF`? 

Thanks!